vercel.json

```diff
@@ -119,6 +119,11 @@
       "source": "/state-tunnel(/)?",
       "destination": "https://github.com/ionic-team/stencil-state-tunnel",
       "statusCode": 301
+    },
+    {
+      "source": "/framework-bindings",
+      "destination": "/docs/overview",
```

Comment:
This is the Frameowrk Overview page - which is...odd. I'll deal with this in a separate PR